### PR TITLE
fix copy-paste oversight in DielectricBuilder.from_file()

### PR DESCRIPTION
**Pull request by janosh**
`DielectricBuilder`'s `from_file` doc string mentions `MaterialsEhullBuilder`.

Btw, this project states support for Python 3.6+ but still uses Python 2 syntax in a lot of places like

```diff
- super(FeffCalcDb, self).__init__(host, port, database, collection,
-                                  user, password, **kwargs)
+ super().__init__(host, port, database, collection, user, password, **kwargs)
```

Happy to include the changes from running `pyupgrade --py36-plus *.py` in this PR, if you want.

### CODE REVIEW

**Comment by utf** [comment]
> Happy to include the changes from running pyupgrade --py36-plus *.py in this PR, if you want.

Thanks! Yes please go ahead and run.

**Comment by janosh** [comment]
@utf `pyupgrade` changed rather a lot of files (139 to be exact). 😅

Let me know if that's too much and you would like me to revert.

Btw, I tried to instal the `pre-commit` hooks in this repo before committing which crashed with

```txt
The 'rev' field of repo 'https://github.com/psf/black' appears to be a mutable reference (moving tag / branch).  Mutable references are never updated after first install and are not supported.  See https://pre-commit.com/#using-the-latest-version-for-a-repository for more details.  Hint: `pre-commit autoupdate` often fixes this.
Updating git@github.com:humitos/mirrors-autoflake.git ... An unexpected error has occurred: CalledProcessError: command: ('/usr/local/bin/git', 'fetch', 'origin', 'HEAD', '--tags')
return code: 128
expected return code: 0
stdout: (none)
stderr:
    git@github.com: Permission denied (publickey).
    fatal: Could not read from remote repository.
    
    Please make sure you have the correct access rights
    and the repository exists.
```

So I updated the links to the `black` and `autoflake` repos:

https://github.com/hackingmaterials/atomate/pull/640/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9R1-R8

After this `pre-commit install` succeeded and the formatting hooks made a bunch more changes before allowing the commit to go through. There are, however, a lot of linter errors remaining which I had to ignore in order to commit.

```
atomate-upstream git:(main) ✗ ga . && gc -m 'changes from pyupgrade --py36-plus *.py'
[WARNING] The 'rev' field of repo 'https://github.com/psf/black' appears to be a mutable reference (moving tag / branch).  Mutable references are never updated after first install and are not supported.  See https://pre-commit.com/#using-the-latest-version-for-a-repository for more details.  Hint: `pre-commit autoupdate` often fixes this.
autoflake................................................................Passed
black....................................................................Passed
Check Yaml...............................................................Passed
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
Flake8...................................................................Failed
- hook id: flake8
- exit code: 1

atomate/vasp/builders/tasks_materials.py:112:13: E722 do not use bare 'except'
atomate/vasp/builders/tasks_materials.py:146:9: E722 do not use bare 'except'
atomate/vasp/workflows/base/exchange.py:2:1: F401 'atomate.vasp.powerups.add_tags' imported but unused
atomate/vasp/workflows/base/exchange.py:2:1: F401 'atomate.vasp.powerups.add_modify_incar' imported but unused
atomate/vasp/workflows/base/exchange.py:2:1: F401 'atomate.vasp.powerups.add_wf_metadata' imported but unused
atomate/vasp/workflows/base/exchange.py:2:1: F401 'atomate.vasp.powerups.add_common_powerups' imported but unused
atomate/vasp/workflows/base/exchange.py:17:1: E402 module level import not at top of file
atomate/vasp/workflows/base/exchange.py:17:1: F401 'pymatgen.analysis.magnetism.analyzer.CollinearMagneticStructureAnalyzer' imported but unused
atomate/vasp/workflows/base/exchange.py:17:1: F401 'pymatgen.analysis.magnetism.analyzer.MagneticStructureEnumerator' imported but unused
atomate/vasp/workflows/base/exchange.py:84:13: E722 do not use bare 'except'
atomate/vasp/workflows/tests/test_bulk_modulus_workflow.py:189:25: E722 do not use bare 'except'
atomate/vasp/firetasks/lobster_tasks.py:267:26: E711 comparison to None should be 'if cond is not None:'
atomate/utils/testing.py:50:13: E722 do not use bare 'except'
atomate/vasp/submission_filter.py:87:25: E722 do not use bare 'except'
atomate/vasp/fireworks/tests/test_exchange.py:32:9: F841 local variable 'new_fw_spec' is assigned to but never used
atomate/lammps/fireworks/core.py:123:36: F821 undefined name 'Topology'
atomate/qchem/workflows/base/fragmentation.py:104:23: E711 comparison to None should be 'if cond is not None:'
atomate/vasp/builders/file_materials.py:59:25: E722 do not use bare 'except'
atomate/lammps/tests/test_drone.py:36:26: F821 undefined name 'LammpsInputSet'
atomate/lammps/tests/test_drone.py:44:23: F821 undefined name 'LammpsLog'
atomate/vasp/builders/materials_ehull.py:99:13: E722 do not use bare 'except'
atomate/vasp/firetasks/tests/test_exchange.py:8:1: F401 'atomate.vasp.firetasks.exchange.HeisenbergConvergence' imported but unused
atomate/vasp/firetasks/tests/test_exchange.py:52:9: F841 local variable 'new_fw_spec' is assigned to but never used
atomate/qchem/drones.py:98:126: E501 line too long (130 > 125 characters)
atomate/qchem/drones.py:269:49: E711 comparison to None should be 'if cond is not None:'
atomate/qchem/drones.py:293:44: E711 comparison to None should be 'if cond is not None:'
atomate/qchem/drones.py:374:126: E501 line too long (142 > 125 characters)
atomate/vasp/builders/bandgap_estimation.py:62:13: E722 do not use bare 'except'
atomate/vasp/fireworks/tests/test_nmr.py:7:1: F403 'from atomate.vasp.fireworks.nmr import *' used; unable to detect undefined names
atomate/vasp/fireworks/tests/test_nmr.py:39:26: F405 'NMRFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.nmr
atomate/vasp/fireworks/tests/test_nmr.py:44:18: F405 'NMRFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.nmr
atomate/vasp/fireworks/tests/test_nmr.py:49:18: F405 'NMRFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.nmr
atomate/vasp/fireworks/tests/test_nmr.py:52:13: F405 'NMRFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.nmr
atomate/vasp/fireworks/tests/test_nmr.py:58:18: F405 'NMRFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.nmr
atomate/vasp/fireworks/tests/test_nmr.py:62:13: F405 'NMRFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.nmr
atomate/vasp/fireworks/tests/test_nmr.py:65:39: F405 'NMRFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.nmr
atomate/vasp/firetasks/parse_outputs.py:185:21: F524 '...'.format(...) is missing argument(s) for placeholder(s): key
atomate/vasp/firetasks/parse_outputs.py:688:9: E722 do not use bare 'except'
atomate/vasp/firetasks/parse_outputs.py:1191:9: E722 do not use bare 'except'
atomate/vasp/fireworks/exchange.py:4:1: F401 'atomate.vasp.firetasks.exchange.HeisenbergConvergence' imported but unused
atomate/vasp/fireworks/exchange.py:59:9: F841 local variable 'additional_fields' is assigned to but never used
atomate/vasp/fireworks/exchange.py:108:9: F841 local variable 'additional_fields' is assigned to but never used
atomate/vasp/firetasks/neb_tasks.py:120:17: E722 do not use bare 'except'
atomate/vasp/firetasks/neb_tasks.py:150:13: E722 do not use bare 'except'
atomate/vasp/firetasks/neb_tasks.py:308:9: E722 do not use bare 'except'
atomate/vasp/firetasks/neb_tasks.py:356:9: E722 do not use bare 'except'
atomate/vasp/builders/tags.py:85:13: E722 do not use bare 'except'
atomate/vasp/builders/tags.py:120:9: E722 do not use bare 'except'
atomate/utils/utils.py:73:5: E722 do not use bare 'except'
atomate/utils/utils.py:337:5: E722 do not use bare 'except'
atomate/vasp/firetasks/tests/test_lobster_tasks.py:103:9: E722 do not use bare 'except'
atomate/vasp/firetasks/tests/test_lobster_tasks.py:118:9: E722 do not use bare 'except'
atomate/vasp/firetasks/tests/test_lobster_tasks.py:136:9: E722 do not use bare 'except'
atomate/qchem/firetasks/critic2.py:46:21: E711 comparison to None should be 'if cond is None:'
atomate/qchem/firetasks/critic2.py:93:21: E711 comparison to None should be 'if cond is None:'
atomate/vasp/workflows/tests/test_vasp_workflows.py:429:48: E712 comparison to True should be 'if cond is True:' or 'if cond:'
atomate/vasp/workflows/tests/test_vasp_workflows.py:440:48: E712 comparison to True should be 'if cond is True:' or 'if cond:'
atomate/vasp/workflows/tests/test_exchange_workflow.py:11:1: F401 'atomate.vasp.firetasks.parse_outputs.MagneticDeformationToDb' imported but unused
atomate/vasp/workflows/tests/test_exchange_workflow.py:11:1: F401 'atomate.vasp.firetasks.parse_outputs.MagneticOrderingsToDb' imported but unused
atomate/vasp/builders/boltztrap_materials.py:57:13: E722 do not use bare 'except'
atomate/vasp/builders/boltztrap_materials.py:161:9: E722 do not use bare 'except'
atomate/vasp/workflows/base/magnetism.py:23:1: E402 module level import not at top of file
atomate/vasp/workflows/base/magnetism.py:25:1: E402 module level import not at top of file
atomate/vasp/workflows/base/magnetism.py:26:1: E402 module level import not at top of file
atomate/vasp/workflows/base/magnetism.py:27:1: E402 module level import not at top of file
atomate/vasp/workflows/base/magnetism.py:28:1: E402 module level import not at top of file
atomate/vasp/workflows/base/magnetism.py:29:1: E402 module level import not at top of file
atomate/vasp/workflows/base/magnetism.py:417:5: F841 local variable 'ordered_energies' is assigned to but never used
atomate/vasp/workflows/base/magnetism.py:438:22: F821 undefined name 'np'
atomate/vasp/workflows/base/magnetism.py:439:21: F821 undefined name 'np'
atomate/vasp/workflows/base/magnetism.py:439:40: F821 undefined name 'np'
atomate/vasp/workflows/base/magnetism.py:462:10: F821 undefined name 'StructureMatcher'
atomate/vasp/workflows/base/magnetism.py:463:66: F821 undefined name 'ElementComparator'
atomate/vasp/workflows/base/magnetism.py:470:9: E722 do not use bare 'except'
atomate/vasp/workflows/base/magnetism.py:490:20: F821 undefined name 'np'
atomate/common/firetasks/glue_tasks.py:380:20: F821 undefined name 'continue_on_missing'
atomate/utils/fileio.py:134:47: E741 ambiguous variable name 'l'
atomate/utils/fileio.py:149:42: E741 ambiguous variable name 'l'
atomate/lammps/firetasks/write_inputs.py:40:44: F821 undefined name 'LammpsData'
atomate/lammps/firetasks/write_inputs.py:83:26: F821 undefined name 'LammpsData'
atomate/lammps/firetasks/write_inputs.py:87:28: F821 undefined name 'LammpsInputSet'
atomate/vasp/workflows/tests/test_nmr.py:80:18: F821 undefined name 'use_custodian'
atomate/qchem/firetasks/run_calc.py:121:22: E711 comparison to None should be 'if cond is None:'
atomate/qchem/firetasks/write_inputs.py:80:21: F821 undefined name 'prev_calc_molecule'
atomate/qchem/firetasks/write_inputs.py:160:21: F821 undefined name 'prev_calc_molecule'
atomate/feff/firetasks/parse_outputs.py:120:13: E741 ambiguous variable name 'l'
atomate/lammps/firetasks/run_calc.py:35:23: F821 undefined name 'LammpsRunner'
atomate/lammps/firetasks/run_calc.py:90:15: F821 undefined name 'PackmolRunner'
atomate/lammps/drones.py:86:22: F821 undefined name 'LammpsInputSet'
atomate/lammps/drones.py:94:35: F821 undefined name 'LammpsDump'
atomate/lammps/drones.py:97:15: F821 undefined name 'LammpsLog'
atomate/lammps/drones.py:104:24: F821 undefined name 'LammpsRun'
atomate/lammps/drones.py:118:59: F821 undefined name 'LammpsRun'
atomate/lammps/drones.py:157:9: E722 do not use bare 'except'
atomate/vasp/fireworks/tests/test_fireworks_core.py:8:1: F403 'from atomate.vasp.fireworks.core import *' used; unable to detect undefined names
atomate/vasp/fireworks/tests/test_fireworks_core.py:39:13: F405 'OptimizeFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:43:26: F405 'StaticFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:45:19: F405 'StaticFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:49:18: F405 'OptimizeFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:50:19: F405 'StaticFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:54:39: F405 'StaticFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:57:18: F405 'OptimizeFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:59:13: F405 'HSEBSFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:62:13: F405 'HSEBSFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:66:18: F405 'HSEBSFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:70:18: F405 'HSEBSFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:74:39: F405 'HSEBSFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:77:18: F405 'OptimizeFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:79:13: F405 'NonSCFFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:82:13: F405 'NonSCFFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:86:19: F405 'NonSCFFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:90:19: F405 'NonSCFFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:94:39: F405 'NonSCFFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:97:26: F405 'DFPTFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:99:19: F405 'DFPTFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:103:18: F405 'OptimizeFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:104:19: F405 'DFPTFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:108:39: F405 'DFPTFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:111:18: F405 'OptimizeFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:113:13: F405 'RamanFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:119:20: F405 'RamanFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:123:20: F405 'RamanFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:127:39: F405 'RamanFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:130:18: F405 'OptimizeFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:132:13: F405 'SOCFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:136:18: F405 'SOCFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:140:18: F405 'SOCFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:144:39: F405 'SOCFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:151:18: F405 'OptimizeFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:153:13: F405 'TransmuterFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:163:18: F405 'OptimizeFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:165:13: F405 'BoltztrapFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:168:20: F405 'BoltztrapFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:172:20: F405 'BoltztrapFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/vasp/fireworks/tests/test_fireworks_core.py:176:39: F405 'BoltztrapFW' may be undefined, or defined from star imports: atomate.vasp.fireworks.core
atomate/lammps/workflows/core.py:65:28: F821 undefined name 'LammpsInputSet'
atomate/lammps/workflows/core.py:146:13: F821 undefined name 'Topology'
```

But that's probably best addressed in another/several PRs?

File: atomate/feff/firetasks/__init__.py [hunk 1]
```diff
@@ -1,5 +0,0 @@
-
-from .glue_tasks import *
-from .parse_outputs import *
-from .run_calc import *
-from .write_inputs import *
```
Comment (utf) [inline atomate/feff/firetasks/__init__.py:5]: Why were these removed?

File: atomate/vasp/workflows/__init__.py [hunk 2]
```diff
@@ -1,2 +0,0 @@
-
-from .presets.core import *
```
Comment (utf) [inline atomate/vasp/workflows/__init__.py:2]: And these? etc

**Review by utf — COMMENTED** [review]
Hi @janosh. The tests are now failing unfortunately. It seems like this is because all imports in `__init__.py` files were removed. In total there are 9 init files affected. 

Would you be able to revert the changes in these files? I think that should fix the failing tests.

File: atomate/feff/firetasks/__init__.py (hunk 1 above)
Comment (janosh) [reply atomate/feff/firetasks/__init__.py]: Oops. didn't look at the changes closely enough. This happened because of the option `--remove-all-unused-imports` flag passed to `autoflake` on [line 6 of `.pre-commit-config.yaml`](https://github.com/hackingmaterials/atomate/blob/main/.pre-commit-config.yaml#L6)

```yml
repos:
  - repo: git@github.com:humitos/mirrors-autoflake.git
    rev: v1.1
    hooks:
    - id: autoflake
      args: [ '--in-place', '--remove-all-unused-imports', '--remove-unused-variable' ]
```

To prevent similar mishaps in the future, I added the `--ignore-init-module-imports` flag. I also took the liberty of changing the repo link to the original `https://github.com/myint/autoflake` which previously linked to a mirror and running `pre-commit update` to update all 3 hooks.

**Comment by utf** [comment]
Thanks, this looks great. I didn't realise `autoflake` had all these features.

**Comment by janosh** [comment]
> Thanks, this looks great. I didn't realise `autoflake` had all these features.

Me neither. Never used it before. :)
